source/diff/diff.cpp

```diff
@@ -748,6 +761,67 @@ void Differ::MatchDebugAndAnnotationInstructions(
   }
 }
 
+std::string Differ::GroupIdsHelperGetName(const IdInstructions& id_to,
+                                          uint32_t id) {
+  return GetSanitizedName(id_to, id);
+}
+
+uint32_t Differ::GroupIdsHelperGetTypeId(const IdInstructions& id_to,
```

Comment:
Same here, why do you need this function?

Reply:
Same response.